.github/workflows/ci.yaml

```diff
@@ -218,9 +218,14 @@ jobs:
       matrix:
         environment: ['production', 'staging', 'develop']
     steps:
-      - id: checkout
-        name: Check out source code
-        uses: actions/checkout@v2
+      # - id: checkout
+      #   name: Check out source code
+      #   uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
This is the dangerous part, it would allow people to run whatever code they want when creating the PR if we allow github actions to run automatically.

If we do restrict github actions we should be good though.